docs/architecture/metadata/workflow.md

```diff
@@ -0,0 +1,29 @@
+## Live Reloading of a Data Page
+
+When modifying a YAML file for a Data page, executing the `etl ... --grapher` command to visualize updates can be inefficient. The following steps outline a streamlined approach for rapid development and testing.
+
+### Example Step
+For illustration, the step `grapher/gcp/2023-07-10/global_carbon_budget` is considered. The corresponding YAML file, `global_carbon_budget.meta.yml`, is assumed to be edited within the grapher channel and previewed on a [staging Data page](http://staging-site-mojmir/admin/datapage-preview/738081).
+
+### Initial Configuration
+- Add `DEBUG=1` to your `.env.myname` file and any other `.env.*` files in use to expedite ETL operations.
```

Comment:
What does DEBUG do? I've just tried with and without, and the difference is small:
```
WITH DEBUG=1
--- Detecting which steps need rebuilding...
OK (5.1s)
--- Running 3 steps:
--- 1. data://garden/gcp/2023-09-28/global_carbon_budget...
OK (17.3s)

--- 2. data://grapher/gcp/2023-09-28/global_carbon_budget...
OK (0.7s)

--- 3. grapher://grapher/gcp/2023-09-28/global_carbon_budget...
...
OK (29.3s)

WITHOUT DEBUG=1
--- Detecting which steps need rebuilding...
OK (4.7s)
--- Running 3 steps:
--- 1. data://garden/gcp/2023-09-28/global_carbon_budget...
OK (18.9s)

--- 2. data://grapher/gcp/2023-09-28/global_carbon_budget...
OK (2.1s)

--- 3. grapher://grapher/gcp/2023-09-28/global_carbon_budget...
...
OK (28.8s)
```

Maybe the global carbon budget is a bad example. I often wanted to see the result of some metadata edits in the data page, but the ETL step was halfway uploading to S3, and I had to wait for it to finish and be executed again automatically. Therefore I stopped using `--watch` and executed the steps always manually. I think `--watch` will be much more useful for shorter steps.

Also, I prefer working on my local grapher because the grapher step runs way faster:
```
WITHOUT DEBUG=1 but on local grapher
--- Detecting which steps need rebuilding...
OK (4.1s)
--- Running 3 steps:
--- 1. data://garden/gcp/2023-09-28/global_carbon_budget...
OK (18.5s)

--- 2. data://grapher/gcp/2023-09-28/global_carbon_budget...
OK (1.9s)

--- 3. grapher://grapher/gcp/2023-09-28/global_carbon_budget...
...
OK (12.4s)
```

Reply:
`DEBUG=1` gets rid of 2s overhead per executed step (updated docs), that's all. It's true that `--watch` won't really work for steps with long execution time. I was using it mostly for editing grapher YAML files.

> Also, I prefer working on my local grapher because the grapher step runs way faster:

Weird, it makes no difference to me. JSON files are uploaded to S3 even if you work locally, so I guess it must be database latency in your case (and it's strange that it's so much).